File: publisher/src/main/java/com/aitor/publisher/dto/IssueResponseTo.java
package com.aitor.publisher.dto;

import lombok.*;
import java.time.LocalDateTime;
import java.util.List;

@Getter
@AllArgsConstructor
public class IssueResponseTo {
    private long id;
    private long userId;
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;
    private List<String> stickers;
}

File: 351002/Yanachkin/Lab6/lab1/Models/DTO/Responses/IssueResponseTo.cs
namespace lab1.Models.DTO.Responses
{
    public class IssueResponseTo
    {
        public long Id { get; set; }
        public string Title { get; set; } = null!;
        public string Content { get; set; } = null!;
        public long EditorId { get; set; }
        public List<long> LabelIds { get; set; } = [];
    }
}
